sources/core/Stride.Core.Mathematics/Color.cs

```diff
@@ -219,7 +219,7 @@ public Color(byte[] values)
         /// Gets or sets the component at the specified index.
         /// </summary>
         /// <value>The value of the alpha, red, green, or blue component, depending on the index.</value>
```

Comment:
This could be re-ordered as well

Reply:
Indeed, 
https://github.com/stride3d/stride/blob/86ee808f19bf96a600d49039fb9eadbb1f77a20f/sources/core/Stride.Core.Mathematics/Color.cs#L200-L202

Should this also be corrected since its the only other occurrence?

Reply:
Yes for the first.
I'll raise an issue about the second one as it looks like a bug given how it is used.